pygmt/helpers/testing.py

```diff
@@ -139,3 +140,40 @@ def wrapper(*args, ext="png", request=None, **kwargs):
         return wrapper
 
     return decorator
+
+
+def download_test_data():
+    """
+    Convenient function to download data files used in tests.
+    """
+    # List of earth_relief suffix to download
+    earth_relief_suffix = [
+        "01d_p",
+        "01d_g",
+        "30m_p",
+        "30m_g",
+        "10m_p",
+        "10m_g",
+        "05m_p",
+        "05m_g",
+    ]
+    # List of tiles of 03s/01s srtm data.
+    # Names like @N35E135.earth_relief_03s_g.nc is for internal use only.
+    # The naming scheme may change. DO NOT USE IT IN YOUR SCRIPTS.
+    earth_relief_tiles = ["@N35E135.earth_relief_03s_g.nc"]
+    # List of cache files
+    cache_files = [
+        "@fractures_06.txt",
+        "@ridge.txt",
+        "@srtm_tiles.nc",  # needed for 03s and 01s relief data
+        "@Table_5_11.txt",
+        "@test.dat.nc",
+        "@tut_bathy.nc",
+        "@tut_quakes.ngdc",
+        "@tut_ship.xyz",
+        "@usgs_quakes_22.txt",
+    ]
+
+    dataset = [f"@earth_relief_{suffix}" for suffix in earth_relief_suffix]
+    dataset.extend(earth_relief_tiles + cache_files)
+    which(fname=dataset, download="a")
```

Comment:
Ok, convert to a single list and rename 'dataset' to 'datasets' as suggested in https://github.com/GenericMappingTools/pygmt/pull/1310/files#r643587926.

```suggestion
    # List of datasets to download
    datasets = [
        # Earth relief grids
        "@earth_relief_01d_p",
        "@earth_relief_01d_g",
        "@earth_relief_30m_p",
        "@earth_relief_30m_g",
        "@earth_relief_10m_p",
        "@earth_relief_10m_g",
        "@earth_relief_05m_p",
        "@earth_relief_05m_g",
        # List of tiles of 03s/01s srtm data.
        # Names like @N35E135.earth_relief_03s_g.nc is for internal use only.
        # The naming scheme may change. DO NOT USE IT IN YOUR SCRIPTS.
        "@N35E135.earth_relief_03s_g.nc",
        # Other cache files
        "@fractures_06.txt",
        "@ridge.txt",
        "@srtm_tiles.nc",  # needed for 03s and 01s relief data
        "@Table_5_11.txt",
        "@test.dat.nc",
        "@tut_bathy.nc",
        "@tut_quakes.ngdc",
        "@tut_ship.xyz",
        "@usgs_quakes_22.txt",
    ]
    which(fname=datasets, download="a")
```